package.json

```diff
@@ -7,7 +7,7 @@
     "test-js": "jest",
     "test-js-all": "yarn run lint-js && yarn run test-js",
     "test-python": "yarn run lint-python && FLASK_DEBUG=0 python3 -m unittest discover tests",
-    "get-licenses": "if [ ! -f webapp/licenses.json ] ; then curl https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json -o webapp/licenses.json; fi",
+    "get-licenses": "if [ ! -f webapp/licenses.json ] ; then curl https://raw.githubusercontent.com/spdx/license-list-data/38ecfa995d4d472090e6730e4edf0296c6a12ed8/json/licenses.json -o webapp/licenses.json; fi",
```

Comment:
@tbille with Docker it will not be executed :|

Reply:
you are right. I updated the way this list is managed, and added some documentation on how to update the list.